### PR TITLE
In 141535089 indicate cross report

### PR DESCRIPTION
**Pull request by sivaOSI**

### CODE REVIEW

File: spec/serializers/screening_serializer_spec.rb [hunk 1]
```diff
@@ -82,6 +90,10 @@
             zip: '12333',
             type: 'Placement'
           },
+          cross_reports: [{
+            agency_type: 'District attorney',
```
Comment (NateOSI) [inline spec/serializers/screening_serializer_spec.rb]: should there be a uniqueness constraint on agency type?

File: db/migrate/20170324173331_create_cross_reports_table.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateCrossReportsTable < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cross_reports do |t|
+      t.string :agency_name
+      t.string :agency_type
+      t.belongs_to :screening, index: true
```
Comment (NateOSI) [inline db/migrate/20170324173331_create_cross_reports_table.rb]: screening should probably have a foreign key constraint

File: app/models/screening.rb [hunk 3]
```diff
@@ -5,8 +5,9 @@ class Screening < ActiveRecord::Base
   has_one :screening_address, inverse_of: :screening
   has_one :address, through: :screening_address
   has_many :participants
+  has_many :cross_reports
```
Comment (NateOSI) [inline app/models/screening.rb]: we should probably add dependent destroys, if this is no longer a mock api

File: db/migrate/20170324173331_create_cross_reports_table.rb (hunk 2 above)
Comment (NateOSI) [inline db/migrate/20170324173331_create_cross_reports_table.rb]: probably need a uniqueness constraint on agency_type and screening_id,
cant have more than one 'District attorney'

File: spec/serializers/screening_serializer_spec.rb (hunk 1 above)
Comment (brunoOSI) [inline spec/serializers/screening_serializer_spec.rb]: It isn't a free form collection, is tied to an UI with checkboxes and is refreshed on every save. Not going to check for dupes at this point.

File: db/migrate/20170324173331_create_cross_reports_table.rb (hunk 2 above)
Comment (brunoOSI) [inline db/migrate/20170324173331_create_cross_reports_table.rb]: Added the constraint.

File: app/models/screening.rb (hunk 3 above)
Comment (brunoOSI) [inline app/models/screening.rb]: Yep, added.

**Pull request merged**